ERCS/erc-7846.md

```diff
@@ -0,0 +1,233 @@
+---
+eip: 7846
+title: Wallet Connection API
+description: Adds JSON-RPC method for requesting wallet connection with modular capabilities.
```

Comment:
Minor nit, but you repeat "wallet connection" from the title in your description. It might be better to reword it so the reader knows what a "wallet connection" _is_.